Makefile

```diff
@@ -131,11 +148,35 @@ lint-static-check:
 lint: lint-static-check
 	$(LINT) run --timeout 5m
 
+.PHONY: multimod-verify
+multimod-verify: | $(MULTIMOD)
+	@echo "Validating versions.yaml"
+	$(MULTIMOD) verify
+
+COREPATH ?= "../opentelemetry-collector-contrib"
+.PHONY: multimod-sync-core
+multimod-sync-core: | $(MULTIMOD)
+	@[ ! -d $COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
+	$(MULTIMOD) verify && $(MULTIMOD) sync -a -o ${COREPATH}
```

Comment:
Do we need to call verify before each of these? If we do need to run it, could that be done with
```make
multimod-sync-core: multimod-verify
```

Reply:
Interesting thought. You are right. It would be more cleaner for me to do in this way.